doc/elements-confidential-transactions.md

```diff
@@ -0,0 +1,480 @@
+# Elements Confidential Transactions
+
+## Audience
+
+This document provides an overview and introduction to
+the Confidential Assets implementation in Elements. A list of relevant
+RPCs is provided as well as a list of references providing further
+information.  
+
+A working knowledge of Bitcoin and Elements, familiarity with Elements
+Remote Procedure Calls (RPCs), and some knowledge of the cryptography
+used in Bitcoin are assumed.
+
+## Overview of Confidential Assets
+
+
+Using Elements, the sender of a transaction can hide the amounts and
+types of assets in a transaction’s outputs, in such a way that: 
+
+1. Only the sender and receiver of the transaction can see the actual
+amounts and types of assets.
+2. A verifier can prove that all assets coming out of a transaction
+went into it. 
+3. The amounts and assets of the outputs may be revealed to a third
+party, by the receiver or by the sender. 
+
+This feature is called Confidential Assets. To create a confidential
+assets transaction, the recipient generates a Confidential Address and
+provides it to the sender. The sender uses the address to create a
+Confidential Transaction, and “blinds” its outputs, obscuring the type
+and quantity of assets while guaranteeing the integrity of the
+transaction. The recipient may later “unblind” and spend the
+transaction’s outputs at will. The unblinding process is called
+“rewinding”, or “rewinding the range proof”, and requires the private
+blinding key of the confidential address. Either the sender or a
+receiver may also share a blinding key with a third party, enabling
+them to view, but not to spend, the transaction’s outputs. 
+
+Confidential Assets does not conceal the transaction ids or indexes on
+the inputs (the transaction graph is public, as it is with Bitcoin). 
+
+A Confidential Transaction must also include an explicit (unblinded)
+fee output, paid in the sidechain’s default asset (L-BTC for Liquid).
+
+## Cryptographic Primitives
+
+Elements' Confidential Assets capability is built on four cryptographic primitives
+implemented in the secpk25k1-zkp C++ library [1]:
```

Comment:
You can actually insert footnote links in markdown.

https://github.blog/changelog/2021-09-30-footnotes-now-supported-in-markdown-fields/

Reply:
thank you fixed in amended commit